packages/waku/src/lib/handlers/handler-dev.ts

```diff
@@ -45,7 +45,7 @@ export function createHandler<
     const mergedViteConfig = await mergeUserViteConfig({
       base: config.basePath,
       optimizeDeps: {
-        include: ['react-server-dom-webpack/client'],
+        include: ['react-server-dom-webpack/client', 'react', 'react-dom'],
```

Comment:
In fact, I used add them, but removed afterwards as they seemed unnecessary, which is wrong.